README.md

```diff
@@ -1,10 +1,22 @@
-# AWS Cloud Control Collection for Ansible
+# Amazon Cloud Collection for Ansible
 The AWS Cloud Control Collection is an experimental alpha collection of generated modules using the Cloud Control API for interacting with AWS Services.
 
-This work is being made available for research purposes on the Cloud Control API and community feedback on the user experience of API generated module like these. Therefore, this content is not intended for production in its current state.
+## Description
+
+AWS Cloud Control Collection's work is being made available for research purposes on the Cloud Control API and community feedback on the user experience of API generated module like these. Therefore, this content is not intended for production in its current state.
```

Comment:
To prevent the last line from being missed due to its length.
```suggestion
AWS Cloud Control Collection's work is being made available for research purposes on the Cloud Control API and community feedback on the user experience of API generated module like these. 
Therefore, this content is not intended for production in its current state.
```